### PR TITLE
♻️ Refactor/OC-681/verbeteringen-footer-template

### PR DESCRIPTION
**Pull request by Ridsko**

### CODE REVIEW

**Comment by Ridsko** [comment]
Tijdelijke oplossing


[🔗](https://github.com/frameless/demodam/issues/166)

File: pwa/static/themes/rotterdam.css [hunk 1]
```diff
@@ -117,9 +117,9 @@
 
   --web-app-color-background-page-top-bar: var(--web-app-color-white);
 
-  --web-app-color-background-footer: #eff4f6;
-  --web-app-color-footer-headings: auto;
-  --web-app-color-footer-links: var(--web-app-color-black);
+  --web-app-color-background-footer: #00811f;
```
Comment (Robbert) [inline pwa/static/themes/rotterdam.css:120]: Gebruik in plaats van de hex kleur liever `var(--rotterdam-color-base-green)`.

Je kunt hier de color tokens opzoeken: https://nl-design-system.github.io/themes/?path=/story/rotterdam-color--page

Dit geldt eigenlijk ook voor alle andere kleuren in `.rotterdam-theme`, zoals `var(--web-app-color-white)`, daar kun je beter `var(--rotterdam-color-base-white)` gebruiken.

**Comment by Ridsko** [comment]
De footer word beter geimplementeerd/gebruikt in deze [PR](https://github.com/OpenCatalogi/web-app/pull/277)

File: pwa/static/themes/rotterdam.css [hunk 2]
```diff
@@ -134,6 +134,7 @@
   --web-app-footer-logo-container-height: 50px;
   --web-app-footer-organization-logo-width: 230px;
   --web-app-footer-organization-logo-src: url("images/LogoRotterdam.svg");
+  --web-app-footer-organization-logo-wit-src: url("images/LogoRotterdamWit.svg");
```
Comment (lencodes) [inline pwa/static/themes/rotterdam.css:137]: Nice addition. However, when adding a new design token to a theme it's _very_ important that we add them to the other themes as well.

In this case: we need to add it to the base theme ".web-app-theme" _and_ the Utrecht theme ".utrecht-theme".

The effect of not adding the new token to the other themes:


image

**Review by lencodes — CHANGES_REQUESTED** [review]
Nice changes to the ThemeSwitcher! Much better this way.

I did leave one comment that we should fix before merging this.

File: pwa/static/themes/rotterdam.css (hunk 2 above)
Comment (Ridsko) [reply pwa/static/themes/rotterdam.css:137]: Sorry this pr should be closed, the improved footer is in this [PR](https://github.com/OpenCatalogi/web-app/pull/277)

**Pull request closed without merging**